examples/twittermap/web/conf/application.conf

```diff
@@ -122,7 +122,7 @@ tweet.url = "http://twitterMapIndex-search-proxy.herokuapp.com/search/tweets?q=%
 # How many recent days of the time range predicate of each heatmap query. (Default: 30)
 heatmap.samplingDayRange = 365
 # How many tweets at most returned for each heatmap query. (Default: 5000)
-# heatmap.samplingLimit = 5000
+heatmap.samplingLimit = 5000
```

Comment:
Do not modify the configuration file. Of course, you can do it locally, this change shouldn't be pushed into our master branch.